beprof/example.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+from beprof.curve import Curve
+import numpy as np
+
+url = 'http://berkeleyearth.lbl.gov/auto/Global/Complete_TAVG_complete.txt'
+data = urllib.request.urlopen(url)
+
+all = np.genfromtxt(data, skip_header=35, skip_footer=2192)
+print(np.shape(all))
+x = np.array(range(0, 1000))
```

Comment:
Again RTFM.
http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.arange.html

Before coding any new line in Python, go to numpy library reference, check which functions are available there and what are they doing. Print such list above your desk. It might help you.